aclk/aclk_otp.c

```diff
@@ -253,9 +283,10 @@ void aclk_get_mqtt_otp(RSA *p_key, char *aclk_hostname, int port, char **mqtt_us
     *mqtt_usr = agent_id;
     agent_id = NULL;
 
-CLEANUP:
+cleanup_resp:
+    https_req_response_free(&resp);
```

Comment:
I got a SIGABRT here (the payload points to invalid memory) when the resp.http_code happens to be 200 but the request has failed

Reply:
yep related to your previous comments... will take over the fix from next PRs

Reply:
should be fixed now